client/src/components/contactUs.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Modal from 'react-responsive-modal';
+import "./contactUs.css";
+
+class ContactUs extends React.Component {
+
```

Comment:
```suggestion
You could follow my signup logic here, for more details, please check my login.jsx
```
`<div style={{ padding: "4%" }}>
        <form onSubmit={this.handleSubmit}>
          <div className="field">
            <label className="label">Email</label>
            <div className="control has-icons-left has-icons-right">
              <input
                className="input is-primary"
                type="email"
                placeholder="Your Email"
                required
                onChange={e => this.handleChange(e, "email")}
                value={this.state.email}
              />

              <span className="icon is-small is-left">
                <i className="fas fa-envelope" />
              </span>
              {this.emailDuplicateError()}
            </div>
          </div>`

Reply:
Ok, do you want me to add it and push the code again?